### PR TITLE
Fixes #5475 Disable DataVirt UI when Komodo server isn't present

### PR DESCRIPTION
**Pull request by rkorytkowski**
Depends on https://github.com/syndesisio/syndesis/pull/5726

https://github.com/syndesisio/syndesis/issues/5475

### CODE REVIEW

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/2545/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

**Comment by mdrillin** [comment]
Agree, I think we should leave config.xxx.json enabled for development

File: app/ui-react/syndesis/config.minishift.json [hunk 1]
```diff
@@ -40,7 +40,8 @@
     "debugDocumentParsing": false
   },
   "datavirt": {
-    "dvUrl": "/vdb-builder/v1/"
+    "dvUrl": "/vdb-builder/v1/",
+    "enabled": 0
```
Comment (gashcrumb) [inline app/ui-react/syndesis/config.minishift.json]: As those folks said, let's keep this enabled in these files, these are only for dev.

File: app/ui-react/syndesis/config.proxy.json [hunk 2]
```diff
@@ -39,7 +39,8 @@
     "debugDocumentParsing": false
   },
   "datavirt": {
-    "dvUrl": "/vdb-builder/v1/"
+    "dvUrl": "/vdb-builder/v1/",
+    "enabled": 0
```
Comment (gashcrumb) [inline app/ui-react/syndesis/config.proxy.json]: And this.

File: app/ui-react/syndesis/config.staging.json [hunk 3]
```diff
@@ -39,7 +39,8 @@
     "debugDocumentParsing": false
   },
   "datavirt": {
-    "dvUrl": "/vdb-builder/v1/"
+    "dvUrl": "/vdb-builder/v1/",
+    "enabled": 0
```
Comment (gashcrumb) [inline app/ui-react/syndesis/config.staging.json]: And this.

**Pull request merged**